moabb/datasets/castillos2023.py

```diff
@@ -0,0 +1,542 @@
+import os.path as osp
+import zipfile as z
+from collections import OrderedDict
+
+import mne
+import numpy as np
+from mne import create_info
+from mne.io import RawArray
+
+from moabb.datasets import download as dl
+from moabb.datasets.base import BaseDataset
+
+
+Castillos2023_URL = "https://zenodo.org/records/8255618"
+
+
+# Each trial contained 12 cycles of a 2.2 second code
+NR_CYCLES_PER_TRIAL = 15
+
+
+def code2array(event_id):
+    codes = OrderedDict()
+    for k, v in event_id.items():
+        code = k.split("_")[0]
+        code = code.replace(".", "").replace("2", "")
+        codes[v - 1] = np.array(list(map(int, code)))
+
+    return codes
+
+
+class BaseCastillos2023(BaseDataset):
+    """c-VEP and Burst-VEP dataset from Castillos et al. (2023)
```

Comment:
You need to put the docstring in the Casitllos*VEP child class, the docstring of BaseCastillos2023 class will only be read by devs and will be less visible on the website.